### PR TITLE
viewer: accessibility: set focus on the main area

### LINKED ISSUE #177: viewer: accessibility: set focus on the main area
When not entering a query in the filter input field, focus should be set by default to the main gallery area (grid) in order to allow the up/down arrows, page up/down and tab buttons to be used to scroll and browse through items.

Currently, the focus seems to be reset to another component each time the main view is updated, which is quite frustrating as the user is forced to click again somewhere in the main area to be able to scroll using the keyboard.

### PR DESCRIPTION
**Pull request by zeroinformatique**
…tion-flow

github: resolves #177

### CODE REVIEW

**Review by pacien — CHANGES_REQUESTED** [review]
The fix done with `moveFocusToContentDiv()` and called when the route changes seems to do the job properly. This is enough to fix the issue described in the linked ticket.

On the other hand, the newly defined `tabindex` properties have a counter-intuitive behaviour and isn't necessary.

Could you drop the `tabindex` part of the changeset and only keep the focus fix on route change?

**Comment by zeroinformatique** [comment]
Like seen together, I split the commit into two branches. We'll study the tabindex order further.
This PR solves the main panel focus display only, now.

File: viewer/src/views/MainLayout.vue [hunk 1]
```diff
@@ -22,7 +22,7 @@
     <ld-title :gallery-title="$galleryStore.galleryTitle" :current-item="$galleryStore.currentItem" />
     <panel-top v-if="isReady" class="layout layout-top" />
     <panel-left v-if="isReady" class="layout layout-left" />
-    <router-view v-if="!isLoading" ref="content" class="layout layout-content scrollbar" />
+    <router-view v-if="!isLoading" ref="content" class="layout layout-content scrollbar" tabindex="01" />
```
Comment (pacien) [inline viewer/src/views/MainLayout.vue:25]: There's still a `tabindex` left here.

**Review by pacien — CHANGES_REQUESTED** [review]

File: viewer/src/views/MainLayout.vue (hunk 1 above)
Comment (zeroinformatique) [reply viewer/src/views/MainLayout.vue:25]: This `tabindex` is required for the main-focus to work properly. This is not a mistake.

**Review by pacien — APPROVED** [review]
Solves the linked issue. LGTM.

**Pull request merged**